app/design/adminhtml/default/default/template/eav/attribute/options.phtml

```diff
@@ -75,6 +75,7 @@
         <div class="hor-scroll">
             <table class="dynamic-grid" cellspacing="0"  cellpadding="0">
                 <tr id="attribute-options-table">
+                    <th style="width:1px">ID</th>
```

Comment:
Works flawlessly, but how about increasing the width a little as it looks too tight, something like `25px`?

<img width="227" alt="Screen Shot 2022-05-17 at 4 32 54 PM" src="https://user-images.githubusercontent.com/67818913/168850429-30d58d2b-3893-49fe-9ada-8cb89d65aea8.png">

Reply:
I'd changed the PR, with a better alignment and spacing (I think) also when numbers became bigger

Example:
<img width="877" alt="Schermata 2022-05-17 alle 18 49 59" src="https://user-images.githubusercontent.com/909743/168878706-6086570f-934d-462f-9e93-19e828d180be.png">